web/server/entrypoint.go

```diff
@@ -164,6 +183,8 @@ func RunServer(ctx context.Context, args []string, _ logging.Logger) (err error)
 	logVersion(logger)
 	versionLogged = true
 
+	logViamEnvVariables(logger)
```

Comment:
can we put this in the logVersion method or slightly refactor the flow so that logViamEnvVariables is also tracked by the defer statement above? maybe change `versionLogged` to `startupInfoLogged` or something